app/models/manageiq/providers/workflows/automation_manager/workflow.rb

```diff
@@ -1,7 +1,15 @@
-class ManageIQ::Providers::Workflows::AutomationManager::Workflow < Workflow
-  def execute(userid: "admin", inputs: {})
+class ManageIQ::Providers::Workflows::AutomationManager::Workflow < ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationScriptPayload
+  def self.create_from_json!(json, **kwargs)
+    json = JSON.parse(json) if json.kind_of?(String)
+    name = json["Comment"]
```

Comment:
Unrelated to this PR, but what happens if the Comment doesn't exists / is blank?